doc/source/parameters/sharp-immersed-boundary-solver/sharp-immersed-boundary-solver.rst

```diff
@@ -134,6 +134,9 @@ This subsection contains the parameters related to the sharp immersed boundary s
     .. note::
 	    The refined cells are all those for which at least one of the degrees of freedom (dof) location satisfies both the ``refine mesh inside radius factor`` and the ``refine mesh outside radius factor`` thresholds. Each cycle of refinement reduces the length of the elements by a factor two.
 
+    .. note::
+        Using values ``refine mesh outside radius factor = 1`` and ``refine mesh inside radius factor = 1`` activates a minimal refinement mode. This enables the solver to select automatically the smallest region near the particle that is permitted by the method.
```

Comment:
Instead of "permitted", I would say that it is the smallest region that guarantees stability of the solution.